jvm-libp2p-minimal/build.gradle.kts

```diff
@@ -9,7 +9,7 @@ import java.nio.file.Files
 import java.nio.file.Paths
 
 // To publish the release artifact to CloudSmith repo run the following :
-// ./gradlew publish -PcloudsmithUser=<user> -PcloudsmithApiKey=<api-key>
+// ./gradlew :jvm-libp2p-minima:publish -PcloudsmithUser=<user> -PcloudsmithApiKey=<api-key>
```

Comment:
typo, should be `:jvm-libp2p-minimal:publish`

Reply:
Fixed